### PR TITLE
add alpha and notalpha modifiers

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

File: modifiers/string.go [hunk 1]
```diff
@@ -89,6 +90,28 @@ func SnakeCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param
...
+}
+
+// NotAlphaCase removes alpha unicode characters. Example: "Everything's here but the letters!" -> "' !"
+// From https://github.com/leebenson/conform
+func NotAlphaCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param string) error {
+	s, ok := v.Interface().(string)
+	if !ok {
+		return nil
+	}
+	v.SetString(regexp.MustCompile(`[\pL]`).ReplaceAllLiteralString(s, ""))
```
Comment (deankarn) [inline modifiers/string.go:111]: Can we make the regexes global for reuse

**Review by deankarn — CHANGES_REQUESTED** [review]
This seems very similar to https://github.com/go-playground/mold/pull/26

one is ascii and the other works with unicode, perhaps the naming can be improved to denote that?

File: modifiers/string.go (hunk 1 above)
Comment (frederikhors) [reply modifiers/string.go:111]: I agree. Done in other PR.

**Comment by frederikhors** [comment]
> This seems very similar to #26
> 
> one is ascii and the other works with unicode, perhaps the naming can be improved to denote that?

I agree. Done in other PR.

**Pull request closed without merging**